### PR TITLE
DBZ-363 Add support for BIGINT UNSIGNED handling for MySQL as a long

### PR DESCRIPTION
**Pull request by vultron81**
By default it BIGINT UNSIGNED will continue to be handled as a byte[]

### CODE REVIEW

File: support/checkstyle/pom.xml [hunk 1]
```diff
@@ -67,4 +67,8 @@
             </exclusions>
         </dependency>
     </dependencies>
-</project>
\ No newline at end of file
+
```
Comment (jpechane) [inline support/checkstyle/pom.xml]: Why this change?

Comment (vultron81) [reply support/checkstyle/pom.xml]: Likely just my IDE

**Comment by vultron81** [comment]
@jpechane Any ideas why my Postgres integration tests are failing?  I barely made any changes to the Postgres code.

**Comment by jpechane** [comment]
LGTM
@vultron81 Thanks!

**Comment by vultron81** [comment]
@jpechane Any idea when this PR will get merged and released in 0.6.1?

**Comment by jpechane** [comment]
@vultron81 @gunnarmorling is on JavaOne this week. We are going to decide relase date (and version) next week after his return.

**Comment by gunnarmorling** [comment]
Looks great, going to merge it. thanks a lot, @vultron81! Would you mind also sending a PR against the website repo for adding the new option to the list of connector properties: https://github.com/debezium/debezium.github.io/blob/develop/docs/connectors/mysql.asciidoc#connector-properties

Thanks!

**Comment by gunnarmorling** [comment]
Rebased, squashed everything into one commit and applied. Thanks again, @vultron81!

**Comment by vultron81** [comment]
Done.
- Ben
